src/Video/Renderer.cpp

```diff
@@ -1,41 +1,34 @@
 #include "Renderer.hpp"
 
 #include <GL/glew.h>
-#include "Lighting/Light.hpp"
-#include "Lighting/Lighting.hpp"
 #include "RenderProgram/StaticRenderProgram.hpp"
 #include "RenderProgram/SkinRenderProgram.hpp"
 #include "RenderSurface.hpp"
 #include "PostProcessing/PostProcessing.hpp"
-#include "PostProcessing/ColorFilter.hpp"
-#include "PostProcessing/FogFilter.hpp"
 #include "PostProcessing/FXAAFilter.hpp"
-#include "PostProcessing/GammaCorrectionFilter.hpp"
-#include "PostProcessing/GlowBlurFilter.hpp"
-#include "PostProcessing/GlowFilter.hpp"
 #include "Texture/Texture2D.hpp"
 #include "Shader/Shader.hpp"
 #include "Shader/ShaderProgram.hpp"
 #include "EditorEntity.vert.hpp"
 #include "EditorEntity.geom.hpp"
 #include "EditorEntity.frag.hpp"
 #include "Geometry/Rectangle.hpp"
-#include "FrameBuffer.hpp"
+#include "Buffer/FrameBuffer.hpp"
+#include "Buffer/StorageBuffer.hpp"
 
 using namespace Video;
 
 Renderer::Renderer() {
     rectangle = new Geometry::Rectangle();
     staticRenderProgram = new StaticRenderProgram();
-    lighting = new Lighting(staticRenderProgram->GetShaderProgram(), rectangle);
-   // skinRenderProgram = new SkinRenderProgram();
-   // postProcessing = new PostProcessing(rectangle);
-   // colorFilter = new ColorFilter(glm::vec3(1.f, 1.f, 1.f));
-   // fogFilter = new FogFilter(glm::vec3(1.f, 1.f, 1.f));
-   // fxaaFilter = new FXAAFilter();
-   // glowFilter = new GlowFilter();
-   // glowBlurFilter = new GlowBlurFilter();
+
+    postProcessing = new PostProcessing(rectangle);
+
+    fxaaFilter = new FXAAFilter();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Member variable 'Renderer::skinRenderProgram' is not initialized in the constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuninitMemberVar)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Member variable 'Renderer::postProcessing' is not initialized in the constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuninitMemberVar)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Member variable 'Renderer::colorFilter' is not initialized in the constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuninitMemberVar)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Member variable 'Renderer::fogFilter' is not initialized in the constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuninitMemberVar)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Member variable 'Renderer::fxaaFilter' is not initialized in the constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuninitMemberVar)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Member variable 'Renderer::glowFilter' is not initialized in the constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuninitMemberVar)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Member variable 'Renderer::glowBlurFilter' is not initialized in the constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuninitMemberVar)